operatorcourier/verify.py

```diff
@@ -0,0 +1,143 @@
+import os
+import logging
+from typing import List, Dict
+import semver
+from operatorcourier.build import BuildCmd
+from operatorcourier.validate import ValidateCmd
+from operatorcourier.errors import OpCourierBadBundle
+from operatorcourier.format import format_bundle
+from operatorcourier import identify
+
+logger = logging.getLogger(__name__)
+FLAT_KEY = '__flat__'
+
+
+class Verification:
+    def __init__(self, source_dir, yamls, ui_validate_io, repository):
+        manifests = self.get_manifests_info(source_dir, yamls)
+        self.nested = len(manifests) > 1
+        self.bundle_dict = None
+        self.validation_dict = \
+            self.get_validation_dict_from_manifests(manifests, ui_validate_io, repository)
+        if not self.nested:
+            self.bundle = format_bundle(self.bundle_dict)
```

Comment:
`bundle` could be a property with content generated on demand/raising exception for nested

Reply:
@MartinBasti So what's the best way to prevent this? Maybe initialize `self.bundle` as `None`?

Reply:
As I told, use property: 
```python
@property
def bundle():
    if self.nested:
        raise ...
   return format_bundle(self.bundle_dict)
```

Reply:
Thanks @MartinBasti. Updated.